src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -307,13 +307,15 @@ class npc_corastrasza : public CreatureScript
 ## npc_iruk
 ######*/
 
-#define GOSSIP_ITEM_I  "<Search corpse for Issliruk's Totem.>"
-
 enum Iruk
 {
+	GOSSIP_MENU_ID_NPC_IRUK                 = 9280,
+	GOSSIP_OPTION_SEARCH_CORPSE             = 0,
+	PC_TEXT_THIS_YOUNG_TUSKARR              = 12585,
```

Comment:
Did you mean `NPC_TEXT_THIS_YOUNG_TUSKARR` ?

Reply:
yep, it misses the N

Reply:
yeah, thanks. It's cp from my old commit :+1: